examples/store_latency.rs

```diff
@@ -54,15 +54,15 @@ unsafe fn boot_store(erase: bool) -> Store<Storage> {
 
 #[derive(Debug)]
 struct StorageConfig {
-    page_size: usize,
+    _page_size: usize,
```

Comment:
If this is not used, let's remove it. I didn't realize it got unused when refactoring last time.

Reply:
Done